thunder/benchmarks/targets.py

```diff
@@ -724,6 +725,35 @@ def backward_fn(result, forward_inputs, output_grads):
     return backward_fn, backward_setup
 
 
+# Thunder executor: RuntimeError: Advanced indexing currently only supports zero or one-dimensional integer tensors,
```

Comment:
Could you file an issue and link it here, please? Thank you.

Reply:
We have https://github.com/Lightning-AI/lightning-thunder/issues/1135 with the same error message
and https://github.com/Lightning-AI/lightning-thunder/issues/764 for a general advanced indexing support.